components/brave_wallet_ui/components/desktop/views/accounts/index.tsx

```diff
@@ -283,13 +306,13 @@ function Accounts (props: Props) {
           </WalletInfoRow>
           <SubviewSectionTitle>{getLocale('braveWalletAccountsAssets')}</SubviewSectionTitle>
           <SubDivider />
-          {userVisibleTokensInfo.filter((token) => !token.isErc721).map((item) =>
+          {accountsTokensList.filter((token) => !token.isErc721).map((item, index) =>
             <PortfolioAssetItem
               spotPrices={transactionSpotPrices}
               defaultCurrencies={defaultCurrencies}
-              key={item.contractAddress}
+              key={item.contractAddress + item.chainId + index}
```

Comment:
Updated!
When I first started working on this PR, `chainId` was not an option in type `BlockchainToken` so had to make the `key` unique somehow since there could be multiple of the same `NativeAsset`, exampel: (ETH on Mainnet, ETH on Ropsten, etc...) which all have a contract address of `''`